packages/mattermost-redux/src/client/client4.ts

```diff
@@ -1926,9 +1926,9 @@ export default class Client4 {
         );
     };
 
-    getPostThread = (postId: string, fetchThreads = true, collapsedThreads = false, collapsedThreadsExtended = false) => {
+    getPostThread = (postId: string, fetchThreads = true, collapsedThreads = false, collapsedThreadsExtended = false, fromCreateAt?: number, fromPost?: string, direction?: 'up'|'down', perPage?: number) => {
         return this.doFetch<PostList>(
-            `${this.getPostRoute(postId)}/thread${buildQueryString({skipFetchThreads: !fetchThreads, collapsedThreads, collapsedThreadsExtended})}`,
+            `${this.getPostRoute(postId)}/thread${buildQueryString({skipFetchThreads: !fetchThreads, collapsedThreads, collapsedThreadsExtended, direction, fromPost, fromCreateAt, perPage})}`,
```

Comment:
Are these query parameters supposed to be camel case? I thought we used snake case for those

Reply:
These are newly added on the server for this particular endpoint.
I'm in the dark on what's the way forward on query parameter naming 😕